corehq/apps/fixtures/templates/fixtures/partials/edit_table_modal.html

```diff
@@ -13,6 +13,12 @@ <h4 class="modal-title">{% trans "Edit Table" %} "<span data-bind="text: tag"></
             <input type="text" class="form-control" data-bind="value: tag, valueUpdate: 'afterkeydown'"/>
           </div>
         </div>
+        <div class="form-group">
```

Comment:
Why isn't this also in the modal to add a new table?

Reply:
Totally unrelated, but I noticed while looking at this that this modal doesn't have a grey backdrop, which deceptively makes it look like you can interact with the rest of the page. Removing `backdrop` from the [modal binding](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js#L343) would fix that (it'll affect other modals, but I think the others should also be showing backdrops).

Reply:
I will admit that I did not test the add to new table modal previously, but the add new table is the same modal as the edit table. I believe this is because it works like:

1. click add new table
2. inserts a row (including modal HTML) into the table
3. modal automatically pops up.

I think this is not a great workflow, but refactoring the whole page/js does seem out of scope for this PR

backdrop change: https://github.com/dimagi/commcare-hq/pull/24295/commits/d0f2ae5c3025d1f60fed7b1a6c5a684c972c4af3

Reply:
Weird, I thought I had tested adding a new table but don't know what I was doing. Thanks for updating the backdrop.